observability/cockpit/how-to/configure-alerts-for-scw-products.mdx

```diff
@@ -0,0 +1,53 @@
+---
+meta:
+  title: How to configure alerts for Scaleway products using Mimir's alert manager
+  description: This page shows you how to configure alerts for Scaleway products using Mimir's alert manager
+content:
+  h1: How to configure alerts for Scaleway products using Mimir's alert manager
+  paragraph: This page shows you how to configure alerts for Scaleway products using Mimir's alert manager
+categories:
+  - observability cockpit
+dates:
+  validation: 2023-10-12
+  posted: 2023-10-12
+---
+
+This page shows you how to configure alerts for Scaleway resources using the [Mimir](/observability/cockpit/concepts/#mimir) alert manager. As Grafana's alert manager does not run indefinitely, we recommend using Mimir's alert manager to receive alerts on your resources.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+  - You have created Scaleway resources to monitor
+  - You have [created your Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/) and selected the **Editor** role
+  - You have [activated managed alerts and created at least one contact point](/observability/cockpit/how-to/activate-managed-alerts/)
+</Message>
+
+1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
+2. Click **Open dashboards** at the top right corner of your screen. You are directed to Grafana.
+3. Log in to Grafana using your credentials.
+4. Click the **Toggle menu** then click **Alerting**.
+5. Click **Alert rules** and **+ Create alert rule**.
+6. Enter a name for your alert.
+7. Select **Mimir or Loki alert** and the data source you want to configure alerts for. For the sake of this documentation we are choosing the **Scaleway Metrics** datasource.
```

Comment:
The UI changes on the current version 10.1.5. 
Now by default the grafana managed alert is used and the user need to click on a button to switch to datasource based alerts.